src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx

```diff
@@ -416,22 +416,25 @@ function AddTaskModal(props) {
                       <label className="form-check-label" htmlFor="notStarted">
                         Not Started
                       </label>
-                    </div>
-                    <div className="form-check form-check-inline">
-                      <input
-                        className="form-check-input"
-                        type="radio"
-                        id="paused"
-                        name="status"
-                        value="Paused"
-                        checked={status === 'Paused'}
-                        onChange={(e) => setStatus(e.target.value)}
-                      />
-                      <label className="form-check-label" htmlFor="paused">
-                        Paused
-                      </label>
-                    </div>
-                    <div className="form-check form-check-inline">
+                  </span>
+                 </div>
+                 <div className="d-flex align-items-center flex-wrap">
+                  <span className="form-check form-check-inline mr-5">
+                        <input
+                          className="form-check-input"
+                          type="radio"
+                          id="paused"
+                          name="status"
+                          value="Paused"
+                          checked={status === 'Paused'}
+                          onChange={(e) => setStatus(e.target.value)}
+                        />
```

Comment:
You have the useState const status and set status and you directly assigning the value. However, what if the user were to input a numeric value or empty string. Hence, we could enclose the logic in a function. In addition, we can use JSDocs With TypeScript. What I suggest is the following:
```suggestion
                         /** @type {[string, function(string):void]} */
  const [status, setStatus] = useState('Started');
  
  **
 
  const handleStatusChange = (event) => {
    const newValue = event.target.value;
    if (newValue.trim() !== '') {
      setStatus(newValue);
    }
```

this way, we take the steps to properly assign the state with the value we desire.

Reply:
Hello @mSharifHub, really appreciate your time reviewing this PR and your excellent suggestions on the styles. I will work on refactoring the styles, however the suggestion on state maintenance is out of scope for this PR.